src/routes/users/handlers.js

```diff
@@ -37,8 +38,8 @@ export async function getPossibleStateCodes(req, res) {
 
 export async function requestVerificationEmail(req, res) {
```

Comment:
This seems like another thing we have to think about before pushing this to prod - are there certain things a user impersonator _shouldn't_ be able to do, like request an email verification?

Reply:
yeah, I raised this concern and a few others at yesterday's standup. I think that along with this specific case, there may also be other things we don't want to allow impersonators to do. I don't have a well-defined list of what those things are yet though, which probably means there are more group conversations to be had

my current view on this is: we allow basically anything if `NODE_ENV` is `development` (because having these powers is very useful to us while building stuff), otherwise we disallow things like this